docs/compatibility.md

```diff
@@ -1,6 +1,6 @@
 # Requests Compatibility Guide
 
-HTTPX aims to be compatible with the `requests` API wherever possible.
+HTTPX aims to be compatible with the `requests` API where we feel it makes sense.
```

Comment:
I know that probably wasn't the intent, but for some reason this phrasing sounds a bit pedantic — like "we" consider that Requests' API was nonsensical in places (when really it's just a result of pioneering the field and having to live on with early decisions, which is understandable).

I also thought of replacing with "wherever sensible", but that seems to carry the same possible misunderstanding.

So I'd be keen to keep the current wording…

```suggestion
HTTPX aims to be compatible with the `requests` API where possible.
```